sdv/sequential/par.py

```diff
@@ -87,12 +87,13 @@ def _get_context_metadata(self):
 
     def __init__(self, metadata, enforce_min_max_values=True, enforce_rounding=False,
                  locales=['en_US'], context_columns=None, segment_size=None, epochs=128,
-                 sample_size=1, cuda=True, verbose=False):
+                 sample_size=1, cuda=True, verbose=False, table_name=None):
```

Comment:
I just realized that this parameter didn't previously exist. I don't think we should add any parameters to the init. We should probably remove it from the single table synthesizers as well or at least consult @npatki 

Instead maybe we set the name on the metadata directly here
https://github.com/sdv-dev/SDV/blob/275955dcaaa81dcf3385b03965839843b80c255b/sdv/multi_table/base.py#L63-L70